### DISCUSSION TITLE
Android APP升级后PRO消失，请协助恢复或退款

### DISCUSSION THREAD

**Discussion by YBian92** [getlantern/lantern-forum-cn#946] [General]
如题
![Lantern_payment](https://github.com/user-attachments/assets/3f331fc2-5eb7-4879-af88-4dc12578e72e)
![Membership_lost](https://github.com/user-attachments/assets/4382584b-fd0f-4927-ac9e-cb5c5fe7b87b)

**Comment by YBian92** [comment]
我的邮箱：yifei.bian@outlook.com，谢谢！

**Comment by MaxMKMax** [comment]
@YBian92 您好. 好的, 我帮您提交问题给技术人员帮您重新升级会专业版.
